ansible/tasks/_inst-foreman-installer.yml

```diff
@@ -0,0 +1,23 @@
+---
+  # Installing Foreman packages
+  - name: Installing Foreman Package
+    yum:
```

Comment:
Please replace the yum module with the package module so it is distro-independent.